### DISCUSSION TITLE
Rectangle Pro stops Photoshop panels from being docked

### DISCUSSION THREAD

**Discussion by sciascia** [rxhanson/RectanglePro-Community#421] [General]
Hi There,

This was the only place I found to post questions/issues etc so sorry if this is the wrong place. 

I've had an ongoing issue where Photoshop's panels no longer dock to the side of the window. After disabling Rectangle Pro, the PS panels dock again so I assume this is an incompatibility between Rectangle and Photoshop?

If so, does anyone know how to fix? I use both apps all day every day.

Cheers, Ben

**Comment by sciascia** [comment]
OK, after more experimentation, it seems it's the 'Snap Areas' feature, if you disable dragging in Settings, you can use both apps. I only use the Rectangle shortcuts anyway, not the dragging features.

Hope that helps someone.

![Rectangle](https://github.com/rxhanson/RectanglePro-Community/assets/875921/d018f670-1456-4ef3-b679-4bc5ade2e414)

**Comment by rxhanson** [comment]
This is the right place for it, thanks for reporting. Here's the issue on Rectangle: https://github.com/rxhanson/Rectangle/issues/1221

TLDR; Unfortunately, this issue is on Adobe's side.

**Comment by amiramir2124** [comment]
https://khanephotoshop.com/

photoshop tutorials
and
Photoshop products
